components/brave_rewards/browser/rewards_service_impl.cc

```diff
@@ -1106,6 +1111,20 @@ void RewardsServiceImpl::LoadActivityInfo(
                      filter.id));
 }
 
+void RewardsServiceImpl::OnPublisherActivityInfoLoaded(
+    ledger::PublisherInfoCallback callback,
+    uint32_t result,
```

Comment:
Is there a reason not to use `ledger::Result` here? Is this something we're doing away with? These calls are initiated from `ledger_client`

Reply:
this is callback from mojo where you have only basic types

Reply:
Correct, but we have lots of `ledger_client` calls from mojo that declare int32 param but use `ledger::Result` param in definition. See `OnWalletProperties` as example

Reply:
`OnWalletProperties` is called from `OnFetchWalletProperties` which cast result and `OnFetchWalletProperties` is actually called from mojo

Reply:
It's also called from `LedgerImpl::OnWalletProperties` (using `ToMojomResult` in mojo client) with no casting back and forth.

Reply:
Not sure how this can be done. Can you please push fix to my PR to address this problem

Reply:
Ok, I see now. From your 2nd comment, this is callback which doesn't have impl for conversion. Unblocked 👍